### DISCUSSION TITLE
FR: Archived entry status

### DISCUSSION THREAD

**Discussion by Kashkin** [craftcms/cms#8672] [Ideas]
We've found that many enterprise or government clients request archive functionality.

For most clients, we are able to push them towards disabling entries, but for large websites with hundreds or thousands of entries, the interface becomes cluttered and their requirements are not being met.

An Archived entry status would address this. Archived entries wouldn't show up on the Entries index page unless specifically filtered to (much like how "Trashed" currently works). An Archive button could be added next to the Delete button on an Entry page. Archived pages are otherwise identical to Disabled pages, and can still be accessed via Share links or Live Preview. It would also allow for template queries to access Archived pages and display them in specific contexts.

#877 would also address this, but I feel that a specific, core entry type for archiving content would be of use to many developers without overcomplicating the interface or code-base.

**Comment by brandonkelly** [comment]
One option is you could disable hard deletion entirely, so soft-deleting elements effectively archives them in perpetuity.

You can do that by setting the [softDeleteDuration](https://docs.craftcms.com/v3/config/config-settings.html#softdeleteduration) config setting to `0` from `config/general.php`.

```php
'softDeleteDuration' => 0,
```

**Comment by kristiansp** [comment]
@brandonkelly That is a decent workaround, but I would also be in favour of a specific archived status like @Kashkin suggests. But as there is an existing archived status, is that only for expired entries, or what does it currently do?

**Comment by brandonkelly** [comment]
The built-in archived status isn’t a fully fleshed-out feature yet, however if an element’s `archived` column (in the `elements` table) is set to `1`, it will stop showing up in queury results unless you call `.archived()` on the element query.

**Comment by kristiansp** [comment]
@brandonkelly Is there currently a way to set an entry to archived status (without updating the DB yourself)? Or is it better to not rely on this for the time being?

**Comment by brandonkelly** [comment]
That’s the only way currently.

**Comment by MatthiasBrodelet** [comment]
I also would like to see something like this implemented.
More specifically for Craft Commerce.
One of our clients has a lot of orders and it would make sense to archive old orders.
